Spotify.py

```diff
@@ -102,31 +102,33 @@ def handle_message(channel, user, message):
                     return "Already used volume up in the last {} minutes. Try again later".format(config.VOLUME_TIME)
 
             if command in COMMAND_DELETE:
-                try:
-                    params = int(params)
-                    if 5 > params > 1 and Spotify.check_user(user):
-                        Spotify.execute_command("del {}").format(params)
-                        USERS[user] = datetime.datetime.now() + datetime.timedelta(minutes=config.WAIT_TIME)
-                    else:
-                        return "Already skipped or deleted a song from the queue in the last {} minutes. " \
-                               "Try again later".format(config.WAIT_TIME)
-                except ValueError:
-                    return "Please choose a number between 1 - {}".format(str(config.QUEUE_LENGTH))
+                if params is not None:
```

Comment:
What happens if params is None?

Reply:
Changed, so it shows the error when params is None